UPGRADING.md

```diff
@@ -68,70 +68,22 @@ The full list of altered methods and their intended replacements can be found be
 
 See the [full documentation](https://docs.bugsnag.com/platforms/android) for more information.
 
-## Client
+## `Client` access
 
-You should should use the static `Bugsnag` interface rather than instantiating `Client` directly. 
+`Client` instances no longer need to be instantiated or referenced directly. 
 
-Many of the previous methods on `Client` should now be called on `Configuration` instead.
-For example, `maxBreadcrumbs` can now only be set on `Configuration` before `Bugsnag` is started:
+Use the `Bugsnag` interface to access all public functionality:
 
-```kotlin
-val config = Configuration("my-api-key")
-config.maxBreadcrumbs = 35
-Client(this, config)
+```diff
+-Client(this, "my-api-key")
++Bugsnag.start(this, "my-api-key")
 ```
 
-The full list of altered methods and their intended replacements can be found below:
-
 | v4.x API  | v5.x API |
 | ------------- | ------------- |
 | `Client#Client(Context, String, boolean)`  | `Bugsnag#start(Context, Configuration)` |
-| `Client#addToTab`  | `Bugsnag#addMetadata(String, String, Object)` and `Configuration#addMetadata` |
```

Comment:
These methods should be kept in the table. When reading this document our users will have no prior knowledge that `Bugsnag` is now favoured over using `Client` and it could seem like no upgrade path has been documented for the `Client` class.